docs/pages/SAPUI5.md

```diff
@@ -1,3 +1,68 @@
 # Consuming SAPUI5 libraries
 
-Coming soon
+!!! info
+    **Make sure you have installed the latest UI5 CLI in version 2.0 or later:**
+
+## Overview
+
+SAPUI5 libraries are hosted on the public npm registry at `registry.npmjs.org`. However, you should not install them using node package managers like npm or Yarn. Instead, please let the UI5 Tooling handle them by following this guide.
+
+## Usage
+Since version 2.0 the UI5 CLI will automatically download all required framework dependencies of a project if they are defined in the corresponding `ui5.yaml` configuration. They will be cached in a `.ui5` directory located in your users home directory. This happens transparently whenever you are executing `ui5 serve` or `ui5 build` commands.
```

Comment:
Since Version 2.0 the UI5 CLI will automatically download all required framework dependencies of a project if they have been defined in the corresponding `ui5.yaml` configuration. They will be cached in a `.ui5` directory located in your users' home directory. This happens transparently whenever you execute the `ui5 serve` or `ui5 build` commands

Reply:
Done